src/pkg/build/pullFurnisher.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+
+package build
+
+import (
+    "strings"
+    "fmt"
+)
+
+// validPullFurnishers contains a list of known PullFurnishers
+var validPullFurnishers = map[string]bool {
```

Comment:
Hey, you need to `go fmt` your code. Most editors should be able to automatically do this every time you save. All code must be formatted before merge!